### PR TITLE
Use bldrs-conway renderer for IFC.  

### PR DESCRIPTION
**Pull request by pablo-mayrgundter**
- Use bldrs-conway-v0.0.1.tgz renderer for IFC.  
- Switch to esbuild compile and bundle
- Support typescript in bable just for jest direct ref of IFCLoader from web-ifc-three.
- Couple fixes for bun, tho not yet using it.

# Demo
```
> yarn install
> yarn build
> yarn serve
yarn run v1.22.19
warning package.json: No license field
$ node --enable-source-maps build/server-bundle.js
Listening on 0.0.0.0:8001
... curl command ...
Conway Web-Ifc Shim v0.1
Geometry Extraction took 703 milliseconds to execute.
Could not get all types of model.
```

```
time curl -f -d "{\"url\": \"https://bldrs.ai/share/v/gh/Swiss-Property-AG/Momentum-Public/main/Momentum.ifc#c:-15.853,5.832,12.224,-4.896,2.027,0.876\"}"     -H 'content-type: application/json'     -o render.png --fail --silent --show-error  -D- http://localhost:8001/render && open render.png
HTTP/1.1 200 OK
X-Powered-By: Express
content-type: image/png
Date: Wed, 18 Oct 2023 04:49:15 GMT
Connection: keep-alive
Keep-Alive: timeout=5
Transfer-Encoding: chunked


real	0m2.044s
user	0m0.004s
sys	0m0.009s
```

image

### CODE REVIEW

File: esbuild/web-ifc-shim-alias-plugin.js
Comment (nickcastel50) [inline esbuild/web-ifc-shim-alias-plugin.js:1]: Looks like there's a hard coded path here.

**Review by nickcastel50 — CHANGES_REQUESTED** [review]

File: esbuild/web-ifc-shim-alias-plugin.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import path from 'node:path'
+
+
+const __dirname = '/Users/pablo/c/b/headless-three/'
```
Comment (nickcastel50) [inline esbuild/web-ifc-shim-alias-plugin.js:4]: Hard coded path here.

File: esbuild/web-ifc-shim-alias-plugin.js
Comment (pablo-mayrgundter) [reply esbuild/web-ifc-shim-alias-plugin.js:1]: Fixed

File: esbuild/web-ifc-shim-alias-plugin.js (hunk 1 above)
Comment (pablo-mayrgundter) [reply esbuild/web-ifc-shim-alias-plugin.js:4]: Done

**Review by pablo-mayrgundter — COMMENTED** [review]
ptal

**Pull request merged**